src/lightly_train/_task_models/dinov3_ltdetr_object_detection/train_model.py

```diff
@@ -191,26 +196,35 @@ def load_train_state_dict(
         Overloads the default implementation to use the task model's loading logic.
         This allows loading weights from an EMA model into the training model.
         """
-        # Initialize the model using the task-model logic
-        incompatible = self.model.load_train_state_dict(
-            state_dict, strict=strict, assign=assign
+        missing_keys, unexpected_keys = self.model.load_train_state_dict(
+            state_dict,
+            strict=strict,
+            assign=assign,
         )
+        if self.ema_model is not None:
+            missing_keys_ema, unexpected_keys_ema = (
+                self.ema_model.model.load_train_state_dict(  # type: ignore
+                    # Copy to avoid assigning the same weights to both models
```

Comment:
Had to update the `load_train_state_dict` logic because I was running into errors